x-pack/plugins/index_management/public/application/components/mappings_editor/__jest__/client_integration/helpers/mappings_editor.helpers.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { registerTestBed, TestBed, nextTick } from '../../../../../../../../../test_utils';
+import { MappingsEditor } from '../../../mappings_editor';
+
+jest.mock('@elastic/eui', () => ({
+  ...jest.requireActual('@elastic/eui'),
+  // Mocking EuiComboBox, as it utilizes "react-virtualized" for rendering search suggestions,
+  // which does not produce a valid component wrapper
+  EuiComboBox: (props: any) => (
+    <input
+      data-test-subj={props['data-test-subj'] || 'mockComboBox'}
+      onChange={async (syntheticEvent: any) => {
+        props.onChange([syntheticEvent['0']]);
+      }}
+    />
+  ),
+  // Mocking EuiCodeEditor, which uses React Ace under the hood
+  EuiCodeEditor: (props: any) => (
+    <input
+      data-test-subj={props['data-test-subj'] || 'mockCodeEditor'}
+      data-currentvalue={props.value}
+      onChange={(e: any) => {
+        props.onChange(e.jsonContent);
+      }}
+    />
+  ),
+}));
+
+const registerActions = (testBed: TestBed<TestSubjects>) => {
```

Comment:
I would interpret this as `createActions` since we are not registering against the TestBed, it is more like a dependency of actions. Just my 5c - no need to action (pun intended)